metabrainz/templates/index/sponsors.html

```diff
@@ -24,8 +24,7 @@ <h1 class="page-title">Sponsors</h1>
       <td><img src="{{ url_for('static', filename='img/logos/sponsors/google.svg') }}"></td>
       <td>
         <b><a href="https://www.google.com/">Google</a></b> has been an extremely supportive
-        sponsor. They've donated over <b>$240,000</b> and they employ our treasurer and
-        secretary, Carol Smith.
+        sponsor. They've donated over <b>$240,000</b>.
       </td>
```

Comment:
$320k so far. I need to actually enter some more entries for google in the donation table.